src/pages/api/graphql.tsx

```diff
@@ -62,36 +53,36 @@ const clientConnection = async () => {
   }
 }
 
-export const apolloServer = new ApolloServer({
+export const apolloServer = new ApolloServer<ServerContext>({
```

Comment:
Data sources are no longer passed in to the server constructor. We also now have a type for the context, `ServerContext`, which is defined in our `types` file.